recipes/fiatlux/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fiatlux" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiatlux-{{ version }}.tar.gz
+  sha256: d1000837ee0449e118e296cf8f96b827bb1dee190ce29db7dc06e35fcf8f5bc1
+
+build:
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ compiler("fortran") }}
+  host:
+    - python
+    - pip
+    - pybind11 >=2.6.0
+    - scikit-build
+    - cmake >=3.15
+  run:
+    - python
+    - libgfortran5
+    - libstdcxx-ng
+    - libgcc-ng
+
+test:
+  imports:
+    - fiatlux
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/scarrazza/fiatlux/
+  summary: API for LUXqed methodology in global PDF fits
+  license: AGPL-3.0-only
+  license_file:
+    - LICENSE
+    - external/yaml-cpp/LICENSE
```

Comment:
Can you please try to build against the `yaml-cpp` package in conda-forge?

Reply:
We have tried however this will introduce an asymmetry with the pypi package. If possible we would prefer to keep the fiatlux code as it is without a special version for conda-forge.

Reply:
This asymmetry is one of the most significant differentiator between the conda(-forge) ecosystem and PyPI. Where on PyPI all non-Python dependencies need to be bundled, in the conda(-forge) world, we have the tooling to unbundle them and keep them in sync across all packages.

Reply:
@xhochy thank you so much for the information. I have just released a new version of fiatlux (0.1.1) that does not depend or require yaml-cpp anymore. This conda-forge recipe was adjusted accordingly and should be for merge.